bokehjs/src/lib/models/annotations/color_bar.ts

```diff
@@ -40,14 +40,22 @@ export class ColorBarView extends BaseColorBarView {
     this.connect(this.model.properties.display_high.change, () => this._metrics_changed())
   }
 
+  get color_mapper(): ColorMapper {
+    // Color mapper that is used to render this colorbar.
```

Comment:
Before this PR, all `ColorMapper` classes render themselves. Now a `StackColorMapper` renders its `alpha_mapper`. This function could be replaced by a virtual function in `ColorMapper` that is overridden in `StackColorMapper`, but the `instanceof` checks are quite common in the color mapper classes so I have followed that approach.